### ISSUE TITLE
Movie Scene Mapping

### ISSUE THREAD

**Issue by cesalo** [Radarr/Radarr#2421]
The issue I'm having is with some sport events like MMA fights. For example: https://www.themoviedb.org/movie/493182-ufc-fight-night-124-stephens-vs-choi
Radarr is not finding any release most likely because they are named just UFC Fight Night 124. I added this to Alternative Titles in TMDB but doesn't seem to make a difference.
How exactly I can use this to tell Radarr to search for "UFC Fight Night 124" additionally to "UFC Fight Night 124 Stephens vs Choi"?
I found the website: https://mappings.radarr.video but not sure how to accomplish this.
Thanks in advance for any assistance.

**Comment by galli-leo** [comment]
Short answer you can't. You might try giving it some time though. TMDB takes time updating.
Long answer we won't provide support for MMA fights or any other sporting events, it's not what Radarr's made for.

**Comment by cesalo** [comment]
fair enough but I just want to understand how it works. For example where is mappings.radarr.video getting  the alternative title / year info? Is it from themoviedb.org? If I made a change and add an alternate title to a movie in themoviedb.org will it be showing in mappings.radarr.video?
I added the alternative titles to this event but I don't see it listed in mappings.radarr.video. Am I doing something wrong? I see that there is a type when adding alternate title in themovidb.org .. Do I need to add an specific type?
